### ISSUE TITLE
Can't get it to work with LG K51

### ISSUE THREAD

**Issue by SpaceSaver** [bkerler/mtkclient#839]
I have an LG K51 and am having issues getting mtkclient to work.  I am grateful for any input on this!  I have so far tried Windows and Ubuntu.  Here's one Ubuntu run.
[log.txt](https://github.com/bkerler/mtkclient/files/13452264/log.txt)

**Comment by SpaceSaver** [comment]
Okay, so I got it to work after a downgrade

**Comment by SpaceSaver** [comment]
I used the LGUP links from here to setup LGUP to downgrade: https://xdaforums.com/t/tutorial-crossflash-bypass-opid-mismatched-error.4345963/

I then downloaded the kdz from here: https://lg-firmwares.com/downloads-file/23880/K500UM10e_09_VZW_US_OP_0825

Then I was able to flash it by holding volume up while plugging in and then with the modified LGUP it just worked with a refurb flash.  Now the device works as expected.  Seems someone was too cheap to use eFuses.
